src/content/core/okteto-variables.mdx

```diff
@@ -197,12 +197,17 @@ USERNAME has a static value, and PASSWORD is dynamically generated.
 Okteto provides a list of variables that serve as feature flags for some of the features we release in our product. Below is a list of those variables that you can use to leverage these newer and experimental features:
 
 - `OKTETO_SMART_BUILDS_ENABLED`: enables the "smart builds" feature that uses a hash of the build context to determine if an image needs to be rebuilt or can be pulled from the registry. This can accelerate build times.
-- `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED`:
-- `OKTETO_KUBERNETES_TIMEOUT`:
-- `OKTETO_HOME`:
-- `OKTETO_FORCE_REMOTE`:
-- `OKTETO_FOLDER`:
-- `OKTETO_REMOTE_CLI_IMAGE`:
+- `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED`: Compose services mounting the same volume will be placed on the same node using Kubernetes's pod affinity. (Defaults to `true`)
+- `OKTETO_KUBERNETES_TIMEOUT`: specifies the timeout while deploying. (defaults to `0`)
+- `OKTETO_FORCE_REMOTE`: if true deploy the repository on the cluster vs locally.
```

Comment:
```suggestion
- `OKTETO_FORCE_REMOTE`: if set to "true", the CLI will run deploy and destroy commands on the cluster instead of running them locally.
```